app/src/main/java/edu/uc/hickmadc/customnotifications/adapters/GroupsAdapter.kt

```diff
@@ -49,10 +49,13 @@ class GroupsAdapter(private val dataSet: ArrayList<Group>) :
         }
     }
 
+    /**
+     * Handles the comparison between groups.
+     */
     private class GroupDiffCallback : DiffUtil.ItemCallback<Group>() {
 
         override fun areItemsTheSame(oldItem: Group, newItem: Group): Boolean {
-            return oldItem.group_id == newItem.group_id
```

Comment:
Naming conventions...

Reply:
Good.   lowerCamelCase is proper in this case, snake_case is not.